android/modules/ui/src/java/ti/modules/titanium/ui/NavigationWindowProxy.java

```diff
@@ -32,6 +32,7 @@ public void open(@Kroll.argument(optional = true) Object arg)
 		// FIXME: Shouldn't this complain/blow up if window isn't specified?
 		if (getProperties().containsKeyAndNotNull(TiC.PROPERTY_WINDOW)) {
 			opened = true;
+			fireEvent(TiC.EVENT_OPEN, null);
```

Comment:
These events will fire if the window is already open/closed. It's best to move the `fireEvent` into the `openWindow` and `closeWindow` methods.

Reply:
but wouldn't that also fire when you run `$.navWindow.openWindow(child)`? Aren't those already captured in the childs window `open` event?